src/encord_active/lib/dataset/outliers.py

```diff
@@ -44,6 +52,16 @@ def get_iqr_outliers(
     moderate_lb, moderate_ub = Q1 - moderate_iqr_scale * IQR, Q3 + moderate_iqr_scale * IQR
     severe_lb, severe_ub = Q1 - severe_iqr_scale * IQR, Q3 + severe_iqr_scale * IQR
 
+    df[_COLUMNS.outliers_status] = OutlierStatus.low.value
+    df.loc[
+        ((severe_lb <= df[_COLUMNS.score]) & (df[_COLUMNS.score] < moderate_lb))
+        | ((severe_ub >= df[_COLUMNS.score]) & (df[_COLUMNS.score] > moderate_ub)),
+        _COLUMNS.outliers_status,
+    ] = OutlierStatus.moderate.value
+    df.loc[
+        (df[_COLUMNS.score] < severe_lb) | (df[_COLUMNS.score] > severe_ub), _COLUMNS.outliers_status
+    ] = OutlierStatus.severe.value
```

Comment:
[nitpick] 💅

when using string enums there is not need to compare string values.

```suggestion
    df[_COLUMNS.outliers_status] = OutlierStatus.low
    df.loc[
        ((severe_lb <= df[_COLUMNS.score]) & (df[_COLUMNS.score] < moderate_lb))
        | ((severe_ub >= df[_COLUMNS.score]) & (df[_COLUMNS.score] > moderate_ub)),
        _COLUMNS.outliers_status,
    ] = OutlierStatus.moderate
    df.loc[
        (df[_COLUMNS.score] < severe_lb) | (df[_COLUMNS.score] > severe_ub), _COLUMNS.outliers_status
    ] = OutlierStatus.severe
```